contracts/staking/ValidatorMessages.sol

```diff
@@ -7,8 +7,8 @@ pragma solidity 0.8.25;
 library ValidatorMessages {
     // The information that uniquely identifies a subnet validation period.
     // The validationID is the SHA-256 hash of the concatenation of the CODEC_ID,
-    // REGISTER_SUBNET_VALIDATOR_MESSAGE_TYPE_ID, and the concatenated ValidationInfo fields.
-    struct ValidationInfo {
+    // REGISTER_SUBNET_VALIDATOR_MESSAGE_TYPE_ID, and the concatenated ValidationPeriod fields.
+    struct ValidationPeriod {
```

Comment:
I like this name. We should adopt this nomenclature whenever we're describing the entity specified by `validationID`.